cbcontainers/state/common/volumes.go

```diff
@@ -0,0 +1,67 @@
+package common
```

Comment:
do you think this logic deserves it's own package, e.g. `common/volumes`?

Reply:
I agree we might want find a better name. currently all the files in that pkg are doing some common mutations, so maybe we need to change the pkg to common/mutations/* wdyt?

Reply:
I think a better separation would be `common/volumes`, `common/otherk8sobject`, etc. 

e.g. separation by k8s object, not by action.

also if we take a look at the consumers:
```go
import "github.com/vmware/cbcontainers-operator/.../common/volumes"

volumes.GetOrCreateVolumeByName()
```

```go
import "github.com/vmware/cbcontainers-operator/.../common/mutations"

mutations.GetOrCreateVolume()
```

both look OK, but the first one can be shortened to:
```go
import "github.com/vmware/cbcontainers-operator/.../common/volumes"

volumes.GetOrCreate()
```

TLDR: I prefer `common/volumes`, but I don't hold that opinion pretty strongly.

Reply:
Hmm I agree we can go for it - so we will have under common
* image
* volumes
* envvars
* probes
* tests
* consts

Reply:
sounds ok, except for:
- what is `tests` for?
- not sure if we need a separate package for `consts`. each type-specific const can be in the type-specific package. the common constants can be in the root.

also - don't feel pressured to do it now. we can just open a Jira about it.

Reply:
cool, i'll open a ticket :)